lib/features/drilldown/DrilldownOverlays.js

```diff
@@ -107,4 +101,13 @@ var getParentChain = function(child) {
   }
 
   return parents.reverse();
-};
+}
+
+
+function planeId(element) {
+  if (is(element, 'bpmn:SubProcess')) {
+    return element.id + '_plane';
+  }
+
+  return element.id;
+}
```

Comment:
Would it make sense to expose this function in a utility Function? We will probably need it in other places as well.

Reply:
We _could_ expose it. However, this is not a safe operation _per se_. If we render the sub-process as a root (only element imported) this naming assumption does not longer hold. 

TLDR: I'd love to continue to improve this in the future, if we have the full picture, not now.

Reply:
Makes sense. Lets not do it now and expose it when we need it :+1: